Flat/src/JetsOps.cc

```diff
@@ -317,6 +325,11 @@ void JetOp::do_execute()
     if (metShift) {
       jets.sort();
       vbf->execute();
+      TLorentzVector vmet;
+      vmet.SetPtEtaPhiM(gt.pfmet[shift], 0, gt.pfmetphi[shift], 0);
+      vmet = vmet + vBadJets;
+      gt.puppimet[shift] = vmet.Pt();
+      gt.puppimetphi[shift] = vmet.Phi();
```

Comment:
Are you sure this is right? If there are no bad jets, you're replacing puppi MET with PF MET. And in any case, we lose the puppi MET.

Reply:
Opss, no, no, we don't want that. It was just a test, I didn't realize it was there.